src/group/controller/collection.ts

```diff
@@ -0,0 +1,28 @@
+import Controller from '@curveball/controller';
+import { Context } from '@curveball/core';
+import { BadRequest } from '@curveball/http-errors/dist';
+import * as userService from '../../user/service';
+import * as hal from '../formats/hal';
+import * as groupService from '../service';
+
+class GroupMemberCollectionControlller extends Controller {
+  async get(ctx: Context) {
+
+    const user = await userService.findById(parseInt(ctx.state.params.id, 10));
+
+    /**
+     * Cheks if user type is a group
+     */
+    const groupUser = await groupService.isGroup(user);
```

Comment:
The variable name suggests that you're getting something back that is a 'user object'